engine/src/main/java/org/terasology/network/internal/NetworkSystemImpl.java

```diff
@@ -797,8 +802,17 @@ private void processRemovedClient(Client client) {
 
     private void processNewClient(NetClient client) {
         client.connected(entityManager, entitySerializer, eventSerializer, eventLibrary);
-        // log after connect so that the name has been set:
-        logger.info("New client connected: {}", client.getName());
```

Comment:
Note that there's nothing wrong with leaving logging in to help debug problems later.  Loggers can be enabled/disabled while the program is running.
If calculating the arguments are expensive, prefix with `if (logger.isInfoEnabled())`
"